xbmc/ServiceManager.h

```diff
@@ -153,6 +157,7 @@ class CServiceManager
   std::unique_ptr<CPlatform> m_Platform;
   std::unique_ptr<PLAYLIST::CPlayListPlayer> m_playlistPlayer;
   std::unique_ptr<CSettings> m_settings;
+  std::unique_ptr<KODI::GAME::CControllerManager> m_controllerManager;
```

Comment:
how about m_gameControllerManager and GetGameControllerManager? At first glance I was confused about this additional manager because I did not notice that this manager is in the namespace GAME.